hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestColStatsRecordWithMetadataRecord.java

```diff
@@ -451,7 +452,11 @@ private HoodieColumnRangeMetadata mergeAndAssert(HoodieColumnRangeMetadata<Compa
     Map<String, Schema> colsToIndexSchemaMap = new HashMap<>();
     colsToIndexSchemaMap.put(colName, Schema.create(schemaType));
 
-    HoodieColumnRangeMetadata actualColumnRange = FileFormatUtils.getColumnRangeInPartition(relativePartitionPath, colName, fileColumnRanges, colsToIndexSchemaMap, V1);
+    // Convert Avro Schema map to HoodieSchema map
+    Map<String, HoodieSchema> hoodieSchemaMap = colsToIndexSchemaMap.entrySet().stream()
+        .collect(Collectors.toMap(Map.Entry::getKey, entry -> HoodieSchema.fromAvroSchema(entry.getValue())));
```

Comment:
Let's simplify this
```
    Map<String, HoodieSchema> hoodieSchemaMap = Collections.singletonMap(colName, HoodieSchema.create(schemaType));
```
Let's also update the method signature to take in HoodieSchemaType instead of an avro type to help break away from the avro dependency